sample-tracking-status-overview-app/src/main/groovy/life/qbic/portal/sampletracking/components/projectoverview/ProjectOverviewView.groovy

```diff
@@ -42,39 +33,20 @@ class ProjectOverviewView extends VerticalLayout{
         titleLabel.addStyleName(ValoTheme.LABEL_LARGE)
         projectGrid = new Grid<>()
 
-        detailsButton = new Button("Show Details",VaadinIcons.INFO_CIRCLE_O)
-        detailsButton.addStyleName(ValoTheme.LABEL_LARGE)
-        detailsButton.enabled = false
-        downloadButton = new Button("Download Batch Data",VaadinIcons.DOWNLOAD)
-        downloadButton.addStyleName(ValoTheme.LABEL_LARGE)
-        downloadButton.enabled = false
-
-        HorizontalLayout buttonLayout = new HorizontalLayout(detailsButton,downloadButton)
-
-        this.addComponents(titleLabel,buttonLayout,projectGrid)
+        this.addComponents(titleLabel, projectGrid)
     }
 
     private void fillGrid(){
-        projectGrid.addColumn({ it.projectCode })
+        projectGrid.addColumn({ it.projectId.projectCode.code})
                 .setCaption("Project Code").setId("ProjectCode")
-        projectGrid.addColumn({it.projectProgress}).setRenderer(progress -> progress, new ProgressBarRenderer())
-        Grid.Column<Project, FailedSamplesRatio> failedSamplesColumn = projectGrid.addColumn({ it.failedSamples })
-                .setCaption("Failed Samples").setId("FailedSamples")
-        //failedSamplesColumn.setRenderer(failedSamples -> failedSamples, new TextRenderer())
-        Grid.Column<Project,Date> lastUpdatedColumn = projectGrid.addColumn({ it.lastUpdate })
-                .setCaption("Last Update").setId("LastUpdate")
-        lastUpdatedColumn.setRenderer(date -> date, new DateRenderer('%1$tY-%1$tm-%1$td'))
-        Grid.Column<Project,String> descriptionColumn = projectGrid.addColumn({ it.projectDescription })
-                .setCaption("Description").setId("Description").setDescriptionGenerator({it.projectDescription})
+        projectGrid.addColumn({ it.projectTitle })
+                .setCaption("Project Title").setId("ProjectTitle")
 
         setupDataProvider()
-
         //specify size of grid and layout
         projectGrid.setWidthFull()
         //todo introduce variable for description column width
```

Comment:
```suggestion
```

maybe it makes sense to limit the width of the project title? I cant estimate how long they will get but I would reuse the limit of the description column limit in this case.